Middleware/BacktraceDebugDataHolder.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace Doctrine\Bundle\DoctrineBundle\Middleware;
+
+use Symfony\Bridge\Doctrine\Middleware\Debug\DebugDataHolder;
+use Symfony\Bridge\Doctrine\Middleware\Debug\Query;
+
+use function array_merge;
+use function array_slice;
+use function debug_backtrace;
+use function in_array;
+
+use const DEBUG_BACKTRACE_IGNORE_ARGS;
+
+/** @psalm-suppress UndefinedClass */
+class BacktraceDebugDataHolder extends DebugDataHolder
+{
+    /** @var string[] */
+    private $connWithBacktraces;
+
+    /** @var mixed[][][] */
```

Comment:
this is `array<string, list<mixed[]>`

Reply:
I think it's `array<string, array<string, mixed>[]>`; replaced